Rewards/Classes/Account/UI/AccountLogin.swift

```diff
@@ -60,10 +61,10 @@ struct AccountLogin: View {
             .background(Rewards.theme.accentColor)
                 .cornerRadius(8)
                 .padding(.top, 32)
-                .alert("Error", isPresented: $showAlert) {
-                } message: {
-                    Button(errorMessage.isEmpty ? "Login Error." : errorMessage, role: .cancel) { }
-                }
+//                .alert("Error", isPresented: $showAlert) {
```

Comment:
I think we should keep checking if the fields are nil before sending the login request...

Reply:
i put back